include/drivers/fsw/driver.h

```diff
@@ -75,7 +75,17 @@ void fsw_ctrl_set_prep_done_reg_if(bool done);
  *
  * @return      void.
  */
-void fsw_ctrl_set_post_work_done(bool override, bool done);
+void fsw_ctrl_set_post_work_done_reg_if(bool override, bool done);
+
+/**
+ * @brief   HW Frequency Switch Get Current MSR Register Interface
+ *
+ * @details Gets the current Mode Switch Register (MSR) in the PHY.
+ *
+ * @return  returns current Mode Switch Register value.
+ * @retval  Possible MSRs values are 0 and 1.
```

Comment:
I know the values are 0 and 1, but should we use WDDR_MSR_0 and WDDR_MSR_1 here to check against as the return vals?

Reply:
We can but I wanted to keep the driver layer a little less abstracted and have the device level work with wddr_msr_t enumeration.

Reply:
Understood, you have identified the possible returns which would indicate which MSR is currently in use, so no issues here.